Xamarin.Forms.Core/RefreshView.cs

```diff
@@ -77,5 +127,13 @@ protected override SizeRequest OnMeasure(double widthConstraint, double heightCo
 		{
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
+
+		protected override void OnPropertyChanged([CallerMemberName] string propertyName = null)
+		{
+			base.OnPropertyChanged(propertyName);
+			if (IsEnabledProperty.PropertyName == propertyName)
```

Comment:
Isn't it easier to read if we did `propertyName == IsEnabledProperty.PropertyName`?